index.adoc

```diff
@@ -1,178 +1,97 @@
 = WildFly Documentation
+:toc:
+:wildfly-latest-version: 35.0.0.Final
+:wildfly-latest-major: 35
 
-== Releases
+== WildFly Application Server
 
-|===
-|WildFly Release | Jakarta EE Version
+WildFly is a powerful, modular, & lightweight application server that helps you build amazing Entreprise Java applications.
 
-|link:35[35.0.0.Final]
-|https://jakarta.ee/specifications/platform/10/apidocs/[Jakarta EE 10]
+Learn more about *link:{wildfly-latest-major}[WildFly {wildfly-latest-version}]* (or link:./archives[previous releases]) and read the link:https://www.wildfly.org/guides/[Guides].
 
+WildFly provides Jakarta EE APIs that are documented in:
 
-|===
+* https://jakarta.ee/specifications/platform/10/apidocs/[Jakarta EE 10]
+* https://jakarta.ee/specifications/platform/11/apidocs/[Jakarta EE 11]
 
-|===
-|Bootable JAR
 
-|link:bootablejar[Bootable JAR]
+== Build Applications With WildFly
 
-|===
+=== WildFly Maven Plugin
 
-|===
-|Galleon
+The WildFly Maven plugin allows to manipulate a WildFly server directly from a Java Maven project.
+Its features include:
 
-|link:galleon[Galleon]
+* Deploy, redeploy or undeploy your application
+* Add resources
+* Execute CLI commands
+* Run a standalone server within Maven
+* Provision & package a WildFly server fit for your application
+* Build a container image with WildFly and your application
 
-|===
+Learn more about the link:wildfly-maven-plugin[WildFly Maven Plugin].
 
-|===
-|WildFly Galleon Plug-ins
+=== Bootable JAR
 
-|link:galleon-plugins[WildFly Galleon Plug-ins]
+Bootable JAR is a Maven plugin to build bootable JAR applications.
```

Comment:
I think this section should move lower and we should say something here about how WildFly Maven Plugin provides this functionality and is our preferred option.